generators/CoreCLRStatefulActor/index.js

```diff
@@ -140,18 +160,20 @@ var ClassGenerator = generators.Base.extend({
     }
 
     this.fs.copyTpl(
-      this.templatePath('service/app/appPackage/servicePackage/Code/entryPoint.sh'),
-      this.destinationPath(path.join(appPackage , appPackagePath, servicePackage, 'Code', 'entryPoint.sh')),
+      this.templatePath('service/app/appPackage/servicePackage/Code/entryPoint'+extension2),
+      this.destinationPath(path.join(appPackage , appPackagePath, servicePackage, 'Code', 'entryPoint'+extension2)),
       {
         serviceProjName : serviceProjName
       } 
     );
-    this.fs.copyTpl(
-      this.templatePath('main/common/dotnet-include.sh'),
-      this.destinationPath(path.join(appPackage, appPackagePath, servicePackage, 'Code', 'dotnet-include.sh')),
-      {
-      }
-    );
+    if(is_Linux){
```

Comment:
Mac?

Reply:
We don't need entrypoint for MAC as discussed.